src/plots/Volume/avtVolumeFilter.C

```diff
@@ -96,16 +87,17 @@ avtVolumeFilter::avtVolumeFilter()
 
 avtVolumeFilter::~avtVolumeFilter()
 {
-    if (primaryVariable != NULL)
+    if (primaryVariable != nullptr)
```

Comment:
:)

Reply:
I just realized a note of explanation might be useful re: Volume plot.
`Volume` used to hold the code for the VTK-8 version of the plot.
`VolumeVTK` held the code for the VTK-9 version.
There was an if-test in plots/CMakeLists.txt to choose which one was built.

I deleted `Volume`, committed. Then renamed `VolumeVTK9` to `Volume`, but didn't make any changes.
In hindsight, this probably could have been handled in a separate PR to make things easier, my apologies.

Reply:
So, do I need to address any of your new Volume plot comments?

Reply:
Ohh that's where they're from. Sorry; sometimes I was just running through code and didn't realize it was from the volume changes.

You're, fine; I'll make them resolved.